src/Card.scss

```diff
@@ -57,10 +66,87 @@
             font-size: 24px;
             font-style: normal;
             font-weight: 200;
-            line-height: 30px;
             letter-spacing: 1.97px;
         }
       }
     }
+
+    //Desktop
+    @media (min-width: 901px) {
+      display: flex;
+      padding: 24px 0px 8px 0px;
+      flex-direction: column;
+      align-items: center;
+      gap: 8px;
+      flex: 1 0 0;
+      align-self: stretch;
+      width: 380px;
+      
+      img {
+        width: 380px;
+        height: 236px;
+      }
+
+      .card-content {
+        padding: 10px;
+        align-items: center;
+  
+        .icon-dish{
+          display: flex;
+          width: 40px;
+          height: 31px;
+          padding: 0px 0.243px 1.32px 0px;
+          justify-content: center;
+          align-items: center;
+        }
+  
+        .icon-rating{
+          width: 216px;
+          height: 40px;
+        }
+  
+        .icon-nis{
+          width: 14px;
+          height: 19px;   
+        }
+  
+        .card-title {
+          text-align: center;
+          font-size: 40px;
+          font-style: normal;
+          font-weight: 400;
+          line-height: 47px; /* 117.5% */
+          letter-spacing: 2.67px;
+        }
+    
+        .card-subtitle {
```

Comment:
i see that .card-subtitle and .card-price contain same fields. 
maybe think of something gives reuse code and not duplicate code?